freqtrade/constants.py

```diff
@@ -533,6 +534,24 @@
         },
         "position_adjustment_enable": {"type": "boolean"},
         "max_entry_position_adjustment": {"type": ["integer", "number"], "minimum": -1},
+        "orderflow": {
+            "type": "object",
+            "properties": {
+                "cache_size": {"type": "number", "minimum": 1, "default": 1000},
+                "max_candles": {"type": "number", "minimum": 1, "default": 1500},
```

Comment:
Are you sure that these 2 default values make sense (also in the docs, obviously)? 

if i read the code right - we tail for 1500 candles - but only keep 1000 candles in the cache, popping the first 500 candles (assuming the default config, obviously).
so from my current understanding, i think they should be aligned